src/main/java/io/github/hypixel_api_wrapper/wrapper/games/bedwars/BedWarsKillCause.java

```diff
@@ -0,0 +1,23 @@
+package io.github.hypixel_api_wrapper.wrapper.games.bedwars;
+
+/**
+ * Different types of kills / deaths that are tracked in BedWars.
+ */
+public enum BedWarsKillCause {
+
+    PROJECTILE("projectile_"),
+    FALL("fall_"),
+    VOID("void_"),
+    ENTITY("entity_attack_"),
+    OVERALL("");
```

Comment:
This is missing quite a few entries (see [here](https://helpch.at/docs/1.8/org/bukkit/event/entity/EntityDamageEvent.DamageCause.html)). I'm gonna end my review here for tonight, but I can add these tomorrow if you'd like.

We might want to use some different names than the keys themselves. For example, "CUSTOM" wouldn't be very clear in this context. Same with things like "SUICIDE", which could be confused with jumping into the void.

Reply:
I looked into the API and only found these types. I don't think hypixel supports all of the types you listed.

Reply:
And you're sure the API keeps track of all of them?

Reply:
Most likely. What I assume they do is listen to bukkit's `PlayerDeathEvent` and get the player's most recent source of damage (see [here](https://www.spigotmc.org/threads/455497)), and then just take its `name().toLowerCase()` and include that in the field. I don't see any reason why they would filter certain causes.

Reply:
Ok. I would double check for all of them because I didn't see many of the listed causes.

Reply:
The top player on the overall bedwars leaderboards (Manhal_IQ_) has all except...
- `BLOCK_EXPLOSION`
- `CUSTOM`
- `FALLING_BLOCK` (anvils)
- `LIGHTNING`
- `MELTING`
- `POSION`
- `STARVATION`
- `SUICIDE`
- `WITHER`

Reply:
I think the cleanest approach would be to include all the "normal" causes in the enum, and then add an "other" constant that adds up the stats for all the other weird causes

I'd consider the normal causes to be
- `VOID`
- `FALL`
- `DROWNING` (water buckets from shop)
- `PROJECTILE` (bow; or guns in armed)
- `ENTITY_ATTACK` (golems/silverfish)
- `ENTITY_EXPLOSION` (tnt/fireballs)

Reply:
Sounds good. I'll add a commit tomorrow.

Reply:
Would it be fine if I included all the kill causes in the same Enum? It would make it so there would be no method overloading.

Reply:
Added the rest of the values to the enum.